_squirtle/mod.ts

```diff
@@ -107,12 +157,12 @@ const runUser = async (
     env = await getEnv()
 
   const { AUTH_TOKEN, ENV } = env
-  const isDev = ENV !== 'prod'
-  const siteUrl = isDev ? 'https://quest-dev.stellar.org' : 'https://quest.stellar.org'
-
   if (!AUTH_TOKEN)
     return console.log(`Please run the <login> command first`);
 
+  const isDev = ENV !== 'prod'
+  const siteUrl = isDev ? 'https://quest-dev.stellar.org' : 'https://quest.stellar.org'
```

Comment:
Makes more sense to me to keep these at the top of the function but if you are going to move them just move them all the way down into the if block where we actually use `siteUrl`